core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3572,6 +3573,38 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  @Override
+  public Map<String, Long> getOperationInfo() {
+    MetricRegistry mr = MetricsSystem.METRIC_REGISTRY;
+    Map<String, Counter> counters = mr.getCounters((name, metric) -> !(name.endsWith("Ops")));
```

Comment:
this seems fragile - if we ever add a new counter that isn't an operation, it will automatically get included here

Reply:
Changed to getMetrics() at the thrift endpoint and divide metrics info into operation and rpc invocation in the shell. Only show certain metric properties.